src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/Term.kt

```diff
@@ -6,4 +6,5 @@ data class Term(
   val weeks: Int? = null,
   val days: Int? = null,
   val hours: Int? = null,
+  val prisonTermCode: String? = null,
```

Comment:
Part of me feels like just `code` is fine since we're within a `Term`, not sure...

Reply:
Yeah I can see that..

Reply:
Maybe the "prison" part adds valuable context? 
Assume careful consideration went into this naming but let me take a note and we can discuss afterwards as a refactor